apps/total-typescript/src/module-builder/video-uploader.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
-import {useFileChange} from './use-file-change'
-import {uploadToS3} from './upload-file'
+import {uploadToS3} from '@skillrecordings/skill-lesson/lib/upload-file'
+import {useFileChange} from '@skillrecordings/skill-lesson/lib/use-file-change'
 
 const VideoUploader = () => {
```

Comment:
@vojtaholik goal is to make this a core ui component, but I'd like to have it handle multiple files at once and do a queue.